packages/coinstac-ui/package.json

```diff
@@ -91,7 +95,8 @@
     "clean:db": "node scripts/clean-db.js",
     "docs": "mkdir -p docs && jsdoc -t ./node_modules/minami -d docs -R README.md -r src/",
     "test": "nyc --reporter=lcov --check-coverage --lines=1 --functions=1 --branches=1 babel-node test/",
-    "watch": "webpack-dashboard -p 3001 -- node webpack-dev-server.js"
+    "watch": "webpack-dashboard -p 3001 -- node webpack-dev-server.js",
+    "e2e": "mocha --timeout=120000 test/e2e/test.js"
```

Comment:
It’s often nice to namespace testing scripts with `test`, so this should be `test:e2e`. See [coins-pr-site’s scripts](https://github.com/MRN-Code/coins-pr-site/blob/dacdab3de4c02d4f1ec402fef15c8e83f5697f19/package.json#L102-L108) for a good example. I also like to alphabetize the `scripts` properties to match how npm automatically alphabetizes `dependencies` and `devDependencies`.